### ISSUE TITLE
Ugly text shadow in beige.css on diagonals in AA fonts

### ISSUE THREAD

**Issue by reagle** [hakimel/reveal.js#1204]
beige.css has a really complex text shadow that renders AA font diagonals poorly IMHO. I've attached a screen shot. I note that absence of text shadows in newer styles but do appreciate them in case one overlays a background of the same color.

![ugly-shadow-aa](https://cloud.githubusercontent.com/assets/596814/7158937/ac5d8028-e34c-11e4-9abd-09b9dc3d7c16.png)

I find this much simpler shadow preferable:

```
*.reveal h1 {
    color: rgba(0,0,0,1.0);
    text-shadow: 0px 0px 10px rgba(255,255,255,0.5);
}
```

**Comment by hakimel** [comment]
Yeah, that old shadow style was way over the top. I tried to go much simpler with the new themes to make them more flexible. If we're changing this for existing themes, I think the shadow should be removed altogether.

**Comment by reagle** [comment]
They are still useful for changing backgrounds (e.g., dark text on dark background/image) but I find you can still get away with something minimal.

**Comment by reagle** [comment]
This is what I use for backgrounds:

```
*.reveal section[data-background] *,
*.reveal section[data-background-iframe] * {
    text-shadow: 2px 2px 4px rgba(190,190,190,1.0);
}
```
